### PR TITLE
Better Chinese Translation

### PR DESCRIPTION
**Pull request by l89669**

### CODE REVIEW

File: MinecraftClient/Protocol/ProtocolHandler.cs [hunk 1]
```diff
@@ -297,7 +299,7 @@ public static LoginResult GetTokenValidation(SessionToken session)
             {
                 string result = "";
                 string json_request = "{\"accessToken\": \"" + JsonEncode(session.ID) + "\", \"clientToken\": \"" + JsonEncode(session.ClientID) + "\" }";
-                int code = DoHTTPSPost("authserver.mojang.com", "/validate", json_request, ref result);
+                int code = DoHTTPSPost("summermc.cc", "/api/yggdrasil/validate", json_request, ref result);
```
Comment (TheSnoozer) [inline MinecraftClient/Protocol/ProtocolHandler.cs]: What does this have to do with a "better translation"?
Why is it needed anyways?

Comment (l89669) [reply MinecraftClient/Protocol/ProtocolHandler.cs]: I'm sorry, only the first commit is for this pull request, I forgot to switch branch so there's other weired commits.
